components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/session/extender/request/SessionExtenderRequestFactoryTest.java

```diff
@@ -54,7 +53,8 @@
 /**
  * Unit test cases for SessionExtenderRequestFactory.
  */
-public class SessionExtenderRequestFactoryTest extends PowerMockTestCase {
+
```

Comment:
Remove redundant line

Reply:
Addressed with commit 12e680f42aedfa730f6f23a0c28f99c8a56868da in https://github.com/wso2/carbon-identity-framework/pull/3774 PR